### ISSUE TITLE
FormatErrorMessage method is being called only once

### ISSUE THREAD

**Issue by idoroshenko** [rpgkaiser/FoolProof.Core#29]
Hey,

I'm getting a format template string in the response JSON without actual values filled, e.g. "{0} is required due to {1} being not equal to {2}". This happens from the second API call. When I make an initial request right after the app starts it works well.

Thanks
Ivan

**Comment by rpgkaiser** [comment]
Hi @idoroshenko,

PRs approved. Thanks for your help.

**Comment by rpgkaiser** [comment]
I'll deploy a new version of the nuget package ASAP.

**Comment by idoroshenko** [comment]
Hey, thank you so much. It's my pleasure to help. Do you want me to suggest some new features and attributes that I found missing and implemented myself? I can create another PR for that.

**Comment by rpgkaiser** [comment]
Sure @idoroshenko, any collaboration is very welcome.

**Comment by rpgkaiser** [comment]
New package version [(1.2.0)](https://www.nuget.org/packages/FoolProof.Core/1.2.0) released.
